### ISSUE TITLE
Default ACT config causes model init failure

### ISSUE THREAD

**Issue by qgallouedec** [huggingface/lerobot#102]
```python
from lerobot.common.policies.act.configuration_act import ActionChunkingTransformerConfig
from lerobot.common.policies.act.modeling_act import ActionChunkingTransformerPolicy

configuration = ActionChunkingTransformerConfig()
model = ActionChunkingTransformerPolicy(configuration)
```

```
Traceback (most recent call last):
  File "/Users/quentingallouedec/lerobot/p.py", line 6, in <module>
    model = ActionChunkingTransformerPolicy(configuration)
  File "/Users/quentingallouedec/lerobot/lerobot/common/policies/act/modeling_act.py", line 75, in __init__
    self.normalize_inputs = Normalize(cfg.input_shapes, cfg.normalize_input_modes, dataset_stats)
  File "/Users/quentingallouedec/lerobot/lerobot/common/policies/normalize.py", line 101, in __init__
    stats_buffers = create_stats_buffers(shapes, modes, stats)
  File "/Users/quentingallouedec/lerobot/lerobot/common/policies/normalize.py", line 31, in create_stats_buffers
    shape = tuple(shapes[key])
KeyError: 'observation.image'
```

**Comment by alexander-soare** [comment]
Thank you for reporting @qgallouedec ! See the linked fix PR.

**Comment by aliberts** [comment]
Closed with #113
